tests/platform_tests/sfp/test_show_intf_xcvr.py

```diff
@@ -57,8 +71,28 @@ def test_check_sfpshow_eeprom(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
     parsed_eeprom = parse_eeprom(sfp_eeprom["stdout_lines"])
     for intf in dev_conn:
         if intf not in xcvr_skip_list[duthost.hostname]:
-            assert intf in parsed_eeprom, "Interface is not in output of 'sfputil show eeprom'"
-            assert parsed_eeprom[intf] == "SFP EEPROM detected"
+            assert intf in parsed_eeprom, (
+                "Interface '{}' is not in output of 'sfputil show eeprom'. "
+                "Verify that the SFP module is properly seated. "
+                "Run 'sfputil show eeprom' on the DUT and check if interface '{}' appears. "
+                "Inspect DUT logs for related errors and confirm the interface '{}' is included "
+                "in the testbed configuration."
+            ).format(
+                intf, intf, intf
```

Comment:
why print the intf 3 times, I gave the comment before, please go through all your diff to make sure don't print the  same variable for several times .